python_legacy/iceberg/core/base_transaction.py

```diff
@@ -60,13 +60,13 @@ def __init__(self, ops, start):
     def table(self):
         return self.transaction_table
 
-    def check_last_operation_commited(self, operation):
+    def check_last_operation_committed(self, operation):
```

Comment:
So while this is a typo, it's also a breaking change to the API.

Since we might use some of this code in the new `python` library, may I suggest to add a note above each saying that they are misspelled? That way, if anybody goes to copy this code to the new `python` library / API, they will update it.

But I don't think it's a good idea to introduce this breaking change, as people likely rely on the (misspelled) name at present.

Reply:
@kbendick 

> may I suggest to add a note above each saying that they are misspelled?

Yes 

>  That way, if anybody goes to copy this code to the new python library / API, they will update it.

If then, do I close this PR, and you are going to comment about it on another PR?

Reply:
You can just add a comment above each, in this PR if you'd like.

Like
```python
# NOTE: function name has typo in the word `comitted`. Kept for backwards compatability in legacy python API.
def check_last_operation_commited(self, operation):
  pass
```

Then, if somebody goes to copy this code to the new python API, that note will come up during code review.

Reply:
> You can just add a comment above each, in this PR if you'd like.
> 
> Like
> 
> ```python
> # NOTE: function name has typo in the word `comitted`. Kept for backwards compatability in legacy python API.
> def check_last_operation_commited(self, operation):
>   pass
> ```
> 
> Then, if somebody goes to copy this code to the new python API, that note will come up during code review.

@kbendick  
Thank you for teaching me good points I applied your suggestion.
I am reading `python_legacy` module for understanding `Apache Iceberg` .
I will open another PR if there are invalid typos like this following the convention which you taught me above.